### ISSUE TITLE
[Android Bootcamp] Pre-work - Review my app

### ISSUE THREAD

**Issue by tuantranGremsy** [tuantranGremsy/ToDoApp#1]
Hi teacher, my app is complete, please review. This is a submission to respond the basic requirements. I will continue to improve something in the next submission. Thank you very much! 
@coderschoolreview

**Comment by coderschoolreview** [comment]
### Thank you for your Rails submission, tuantranGremsy!

:no_entry_sign: Unfortunately **there are some problems** with your submission:
- I couldn't find a Heroku deploy link listed in your `README.md`. Please note the URL where you have deployed your app.

**Comment by tuantranGremsy** [comment]
Sorry, I don't fully understand. Are there anything wrong? Or did I miss an important thing?

**Comment by tuantranGremsy** [comment]
This is Android pre-work app. It's not for Rails submission, please check. Thank you!

**Comment by harley** [comment]
Hi @tuantranGremsy -- apologies for the confusion! Thanks for your submission for Android. We'll start reviewing the application from next week and you'll get feedback from us soon after that. In the mean time, you are encouraged to enhance submission to make it even more interesting. We will showcase the best submissions when we start the course.

Best,
@coderschoolreview

**Comment by hoanle** [comment]
@tuantranGremsy 

Thanks for the submission

Unfortunately, your app crashes right at the start. The item list for adapter seems to be null. 
Please correct the issue and i will review it again.

**Comment by harley** [comment]
Hi @tuantranGremsy, have you got the chance to update your app for us to review? Please note that the submission deadline is March 1 :)

**Comment by harley** [comment]
Ah I've seen your other submission. All good. Thanks :)

**Comment by coderschoolreview** [comment]
### Thank you for your Android submission, tuantranGremsy!

 :smile: Everything looks good with the format of your submission. We'll be reviewing your submission soon!

**Comment by tuantranGremsy** [comment]
Yes, I have submitted a new one called MyTodo because ToDoApp  was crashed. 
I also got some updates from Mr. hoanle in MyTodo's issue.
According to his suggestions, I will improve something as soon as possible.
Please focus on MyTodo app instead of this TodoApp.
Thank you very much for your reminders. @coderschoolreview  @harley @hoanle
